doc/design/cluster_status.md

```diff
@@ -33,5 +33,5 @@ In summary, the TPR:
 
 - receive spec update (or initialize spec) with a resource version
 - collect cluster status during reconciliation
-- atomically update cluster status with known resource version after reconciliation if there is a status change
+- automatically update cluster status with known resource version after reconciliation if there is a status change
```

Comment:
Revert this change. It should be "atomic"

Reply:
fixed